packages/monitors/src/OptimisticOracleContractMonitor.js

```diff
@@ -185,6 +201,16 @@ class OptimisticOracleContractMonitor {
     this.lastSettlementBlockNumber = this._getLastSeenBlockNumber(latestEvents);
   }
 
+  // Returns helper method for converting collateral token associated with financial contract
+  // to human readable form.
```

Comment:
nit: line length.
```suggestion
 // Returns helper method for converting collateral token associated with financial contract to human readable form.
```

Reply:
bit of a petty this requires two additional sequential async calls but not really any way around it.